services/QuillLMS/client/package.json

```diff
@@ -89,7 +90,7 @@
     "react-dropzone": "^4.2.8",
     "react-html-parser": "^2.0.2",
     "react-i18next": "^8.4.0",
-    "react-markdown": "^2.4.6",
+    "react-markdown": "^2.5.1",
```

Comment:
If we already have `react-markdown` in here we probably don't also need `marked`, right?